pkg/scrape/scrape.go

```diff
@@ -180,7 +180,7 @@ func (sp *scrapePool) reload(cfg *config.ScrapeConfig) {
 	for fp, oldLoop := range sp.loops {
 		var (
 			t       = sp.activeTargets[fp]
-			s       = &targetScraper{Target: t, client: sp.client, timeout: timeout}
+			s       = &targetScraper{Target: t, logger: sp.logger, client: sp.client, timeout: timeout}
```

Comment:
There was a bug here, `logger` was left `nil`, causing Parca to panic as soon as the new scraper tries to log